packages/boba/register/addresses/addressesOptimismMainnet_0x.json

```diff
@@ -1,8 +1,8 @@
 {
   "TK_L1ETH": "0x0000000000000000000000000000000000000000",
-  "TK_L2ETH": "0x0000000000000000000000000000000000000000",
+  "TK_L2ETH": "0x4200000000000000000000000000000000000006",
   "Proxy__L1Teleportation": "0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801",
-  "Proxy__L2Teleportation": "0x2dE73Bd1660Fbf4D521a52Ec2a91CCc106113801",
+  "Proxy__L2Teleportation": "0x0dfFd3Efe9c3237Ad7bf94252296272c96237FF5",
   "L1Teleportation": "0x3f7Da9C51138E0475aA26E80677d27A568cFD6b9",
-  "L2Teleportation": "0x3f7Da9C51138E0475aA26E80677d27A568cFD6b9"
+  "L2Teleportation": "0x670b130112C6f03E17192e63c67866e67D77c3ee"
```

Comment:
does not exist: https://optimistic.etherscan.io/address/0x670b130112C6f03E17192e63c67866e67D77c3ee